src/test/java/donnafin/logic/commands/AppendCommandTest.java

```diff
@@ -0,0 +1,129 @@
+package donnafin.logic.commands;
+
+import static donnafin.testutil.TypicalPersons.GEORGE;
+import static donnafin.testutil.TypicalPersons.getTypicalAddressBook;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.nio.file.Path;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import donnafin.commons.core.types.Money;
+import donnafin.commons.exceptions.IllegalValueException;
+import donnafin.logic.PersonAdapter;
+import donnafin.logic.commands.exceptions.CommandException;
+import donnafin.model.AddressBook;
+import donnafin.model.Model;
+import donnafin.model.ModelManager;
+import donnafin.model.UserPrefs;
+import donnafin.model.person.Asset;
+import donnafin.model.person.Liability;
+import donnafin.model.person.Person;
+import donnafin.model.person.Policy;
+import donnafin.storage.JsonAddressBookStorage;
+import donnafin.storage.JsonUserPrefsStorage;
+import donnafin.storage.StorageManager;
+
+public class AppendCommandTest {
+    private static Asset testAsset;
+    private static Liability testLiability;
+    private static Policy testPolicy;
+
+    private static Set<Asset> combinedAssets;
+    private static Set<Liability> combinedLiability;
+    private static Set<Policy> combinedPolicy;
+
+    @TempDir
+    public Path temporaryFolder;
+    private PersonAdapter personAdapter;
+
+    private final Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs(), null);
+
+    @BeforeEach
+    public void setUp() throws IllegalValueException {
+        JsonAddressBookStorage addressBookStorage =
+                new JsonAddressBookStorage(temporaryFolder.resolve("addressBook.json"));
+        JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(temporaryFolder.resolve("userPrefs.json"));
+        StorageManager storage = new StorageManager(addressBookStorage, userPrefsStorage);
+        Model model = new ModelManager(new AddressBook(), new UserPrefs(), storage);
+        model.addPerson(GEORGE);
+        personAdapter = new PersonAdapter(model, GEORGE);
+
+        try {
+            testPolicy = new Policy("Golden Mile", "AIA",
+                    "$2000", "$100", "$200");
+            testLiability = new Liability("Mortgage debt", "Debt", "$20000",
+                    "23 year loan from DBS Bank.");
+            testAsset = new Asset("Good Class Bungalow", "Property", "$2000000",
+            "Paid in full. No debt.");
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(Money.MESSAGE_CONSTRAINTS);
+        }
```

Comment:
Why are we catching and rethrowing the same exception here? It doesn't make sense in a test i feel.

Reply:
Hmmm I think last time when I didn't surround it in a try-catch block it threw an error but now it doesn't.... I can just remove the try-catch block overall since it will always use a valid input (in this case).